shared/metrics/src/main/java/io/pravega/shared/metrics/StatsLogger.java

```diff
@@ -52,7 +52,7 @@
      * @param valueSupplier the supplier to provide value through get()
      */
     Gauge registerGauge(String name, Supplier<Number> valueSupplier, String... tags);
-    
+
     /**
```

Comment:
Revert changes to this file.

Reply:
I tried to added the tab back, but git removed it for me after a few seconds (when I run "git status") and insist there's no difference. This is due to some git config?

Reply:
Try to replace this with 4 spaces.

Reply:
If you replace with four spaces, then we will still have a change to the file. If we are to change to that line, I'd rather eliminate the spurious tab/spaces altogether rather than replace a tab with spaces.

Reply:
Tried space as well. Seems it is a git feature to remove the trailing spaces automatically on a blank line. @andreipaduroiu - are you ok to leave as it is?

Reply:
If that is indeed the issue, please change it to whatever won't cause git to auto-change it again.

Reply:
fixed now,